fastlane/Fastfile

```diff
@@ -240,6 +273,25 @@ platform :android do
     )
   end
 
+  lane :build_qa do
+    prepare_release
+
+    build_android
+
+    abbreviated_commit_hash = last_git_commit[:abbreviated_commit_hash]
+
+    s3 = Aws::S3::Resource.new(region: ENV['AWS_REGION'])
+
+    apk_obj = s3.bucket(ENV['AWS_BUCKET_NAME']).object("android/#{abbreviated_commit_hash}.apk")
+    apk_obj.upload_file("#{lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]}")
+
+    android_install_path = "https://s3.#{ENV['AWS_REGION']}.amazonaws.com/#{ENV['AWS_BUCKET_NAME']}/android/#{abbreviated_commit_hash}.apk"
+
+    if ENV['MATTERMOST_WEBHOOK_URL']
```

Comment:
same as abot

Reply:
Done, good idea. That new message format looks great.

Reply:
also because icons can actually help here we could use:
http://www.concretesolutions.com.br/blog/wp-content/uploads/2015/04/Android1.png as the icon url for android builds and for ios we can use https://lh3.googleusercontent.com/Nkbo3QohCOU8bGqSYtwB88o03XxUwRAxRHNdXPB9zFvYFzTwD7naYa-GYJaelBp-OIc=w300 or something similar

Reply:
I added the icons and switched the URLs b/c one was 301 and I the other is now hosted by Apple.